src/utils.ts

```diff
@@ -1,81 +1,100 @@
+import * as fs from 'fs';
+import * as core from '@actions/core';
 import * as dockleHelper from './dockleHelper';
 import * as gitHubHelper from './gitHubHelper';
 import * as inputHelper from './inputHelper';
 import * as trivyHelper from './trivyHelper';
 
 export function getCheckRunPayloadWithScanResult(trivyStatus: number, dockleStatus: number): any {
-    const headSha = gitHubHelper.getHeadSha();
-    const checkConclusion = getCheckConclusion(trivyStatus, dockleStatus);
-    const checkSummary = getCheckSummary(trivyStatus, dockleStatus);
-    const checkText = getCheckText(trivyStatus, dockleStatus);
+  const headSha = gitHubHelper.getHeadSha();
+  const checkConclusion = getCheckConclusion(trivyStatus, dockleStatus);
+  const checkSummary = getCheckSummary(trivyStatus, dockleStatus);
+  const checkText = getCheckText(trivyStatus, dockleStatus);
 
-    const checkRunPayload = {
-        head_sha: headSha,
-        name: `[container-scan] ${inputHelper.imageName}`,
-        status: "completed",
-        conclusion: checkConclusion,
-        output: {
-            title: "Container scan result",
-            summary: checkSummary,
-            text: checkText
-        }
+  const checkRunPayload = {
+    head_sha: headSha,
+    name: `[container-scan] ${inputHelper.imageName}`,
+    status: "completed",
+    conclusion: checkConclusion,
+    output: {
+      title: "Container scan result",
+      summary: checkSummary,
+      text: checkText
     }
+  }
 
-    return checkRunPayload;
+  return checkRunPayload;
+}
+
+export function getConfigForTable(widths: number[]): any {
+  let config = {
+    columns: {
+      0: {
+        width: widths[0],
+        wrapWord: true
+      },
+      1: {
+        width: widths[1],
+        wrapWord: true
+      },
+      2: {
+        width: widths[2],
+        wrapWord: true
+      },
+      3: {
+        width: widths[3],
+        wrapWord: true
+      }
+    }
+  };
```

Comment:
NIT: Instead you could use a map function or something right?
Then it could handle expansion later in case.

Reply:
This is not a change that I made in this PR. I just relocated the function to keep all export functions together. We can change this separately.